lib/engine/game/g_1824/step/buy_sell_par_exchange_shares.rb

```diff
@@ -9,6 +9,7 @@ module Step
         class BuySellParExchangeShares < Engine::Step::BuySellParShares
           EXCHANGE_ACTIONS = %w[buy_shares].freeze
           BUY_ACTION = %w[special_buy].freeze
+          PURCHASE_ACTIONS = Engine::Step::BuySellParShares::PURCHASE_ACTIONS + [Action::SpecialBuy]
 
```

Comment:
why is this not a string array as other *_ACTIONS?

Reply:
huh, it's not in the base `BuySellParShares` class either, looks like it's used in a different way than normal, not for the `actions` method

https://github.com/tobymao/18xx/blob/b051e225c83b0d935dc93859ae6169646147de5f/lib/engine/step/buy_sell_par_shares.rb#L16

https://github.com/tobymao/18xx/blob/b051e225c83b0d935dc93859ae6169646147de5f/lib/engine/step/buy_sell_par_shares.rb#L132-L142

https://github.com/tobymao/18xx/blob/b051e225c83b0d935dc93859ae6169646147de5f/lib/engine/step/buy_sell_par_shares.rb#L310-L312